packages/kit/src/exports/public.d.ts

```diff
@@ -797,7 +797,7 @@ export interface LoadEvent<
 	 *
 	 * Be careful not to introduce accidental waterfalls when using `await parent()`. If for example you only want to merge parent data into the returned output, call it _after_ fetching your other data.
 	 */
-	parent(): Promise<ParentData>;
+	parent(): MaybePromise<ParentData>;
```

Comment:
This is also public API. We must never return a `MaybePromise` from a public function (see https://blog.ometer.com/2011/07/24/callbacks-synchronous-and-asynchronous/)